src/main/java/org/klodnicki/controller/MenuController.java

```diff
@@ -0,0 +1,49 @@
+package org.klodnicki.controller;
+
+import org.klodnicki.service.MenuService;
+import org.klodnicki.view.MenuView;
+
+import java.util.Scanner;
+
+public class MenuController {
+    private static final String WELCOME = """
+            Welcome to Library Manager application.
+                        
+            Type a command. For help type "help"
+            """;
+
+    private static final String USER_COMMAND = "Command:";
+
+    private static final String OPERATION_SUCCESS = "Operation succeeded!";
+    private static final String ABORT_OPERATION = "Operation has been canceled. Not found such command.";
+
+    private final MenuView menuView = new MenuView();
+    private final MenuService menuService = new MenuService();
+
+    public void run() {
+        menuService.addMainMenuCommands(this);
+        menuView.update(WELCOME);
+
+        String userInput = askForInput(USER_COMMAND);
```

Comment:
Lines 28-31:
This is incorrect. What you want to achieve here is print either `OPERATION_SUCCESS` or `ABORT_OPERATION`.
Ask yourself now @BartoliniAndBorderCollies, what will happen if `executeCommand` return true? And what will happen when the same command returns false?

SPOILER and answer below:
If that method return `false`, app will correctly print only `ABORT_OPERATION`, but when it returns `true` it'll print both the success and abort messages.